Assets/Scripts/Data/Pickup Pup/Currency/DogFoodData.cs

```diff
@@ -21,7 +21,9 @@ public override Sprite Icon
     {
         get
         {
-            return base.Icon;
+            Sprite icon;
+            spriteDatabase.TryGetSprite("10_DogFood", out icon);
```

Comment:
I'm going to make it a single constant because I think the icon will have its own file name (one that's not shared with the 10 DogFood shop item).